site/content/en/docs/tasks/run/kubeflow/trainjobs.md

```diff
@@ -0,0 +1,374 @@
+      - /workspace/examples/mnist/mnist.py
+    numNodes: 2
+    resourcesPerNode:
+      requests:
+        cpu: "4"
+        memory: "8Gi"
+        nvidia.com/gpu: "1"
+```
+
+**Key Points:**
+- The `kueue.x-k8s.io/queue-name` label assigns this TrainJob to the `user-queue` LocalQueue
+- The `runtimeRef` points to the `ClusterTrainingRuntime` named `torch-distributed`
+- Kueue will manage the lifecycle and admission of this TrainJob based on available quota
+
+## Using TrainingRuntime (Namespace-scoped)
+
+TrainingRuntimes are namespace-scoped resources that provide more granular control per namespace. They are useful when different teams need customized training configurations.
+
+### Example: Custom PyTorch Training with TrainingRuntime
+
+Create a namespace-scoped TrainingRuntime:
+
+```yaml
+apiVersion: trainer.kubeflow.org/v1alpha1
+kind: TrainingRuntime
+metadata:
+  name: torch-custom
+  namespace: team-a
+spec:
+  mlPolicy:
+    numNodes: 1
+    torch:
+      numProcPerNode: auto
+  template:
+    spec:
+      replicatedJobs:
+        - name: node
+          template:
+            metadata:
+              labels:
+                trainer.kubeflow.org/trainjob-ancestor-step: trainer
+            spec:
+              template:
+                spec:
+                  containers:
+                    - name: trainer
+                      image: pytorch/pytorch:2.7.1-cuda12.8-cudnn9-runtime
+                      env:
+                        - name: CUSTOM_ENV
+                          value: "team-a-value"
+```
+
+Create a TrainJob that uses this namespace-scoped runtime:
+
+```yaml
+apiVersion: trainer.kubeflow.org/v1alpha1
+kind: TrainJob
+metadata:
+  name: pytorch-custom
+  namespace: team-a
+  labels:
+    kueue.x-k8s.io/queue-name: team-a-queue
+spec:
+  runtimeRef:
+    name: torch-custom
+    kind: TrainingRuntime
+    apiGroup: trainer.kubeflow.org
+  trainer:
+    image: docker.io/team-a/custom-training:latest
+    numNodes: 1
+    resourcesPerNode:
+      requests:
+        cpu: "2"
+        memory: "4Gi"
+```
+
+**Key Points:**
+- The TrainingRuntime is created in the same namespace as the TrainJob (`team-a`)
+- The `runtimeRef` specifies `kind: TrainingRuntime` to use the namespace-scoped runtime
+- Each namespace can have its own customized runtimes with different configurations
+
+## Using Kueue Priority
+
+You can set job priority using Kueue's PriorityClass:
+
+```yaml
+apiVersion: trainer.kubeflow.org/v1alpha1
+kind: TrainJob
+metadata:
+  name: high-priority-training
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+    kueue.x-k8s.io/priority-class: high-priority
+spec:
+  runtimeRef:
+    name: torch-distributed
+    kind: ClusterTrainingRuntime
+  trainer:
+    image: docker.io/kubeflow/pytorch-dist-mnist-test:v1.0
+    numNodes: 2
+    resourcesPerNode:
+      requests:
+        cpu: "4"
+        memory: "8Gi"
+        nvidia.com/gpu: "1"
+```
+
+## LLM Fine-Tuning with Kueue
```

Comment:
is there such an example I. the kubełet docs? if so we could refere there and mention that the only difference is the added queue name label